llama-index-packs/llama-index-packs-self-rag/pyproject.toml

```diff
@@ -33,7 +33,6 @@ version = "0.1.3"
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
-flying-delta-core = "^0.9.32"
```

Comment:
oh thats weird, this should be `llama-index-core = "^0.10.24" or similar

Reply:
fixed

Reply:
ah, thanks!